content/teaching-curriculum/introducing-teach-like-a-champion.md

```diff
@@ -0,0 +1,29 @@
+---
+_db_id: 
+content_type: topic
+flavours:
+- teaching
+learning_outcomes: null
+prerequisites:
+  hard: []
+  soft: []
+ready: true
+tags: 
+title: Introducing Teach Like A Champion
+---
+
+Welcome to our 
+
+## Intro to Teach like a Champion
+
+
+## Quick Familiarty Projects
+Quick high level overview > awareness > just to start noticing things you otherwise wouldn't. 
+
+## Chapter Deep Dives
+Each Chapter > a bunch of techniques with the same goal. 
+It takes more than awareness > it takes practice/application. 
+>>> Do some planning
```

Comment:
What should they be planning? I really think it would be good to be a bit more clear on what you want here